pkg/framework/framework.go

```diff
@@ -34,6 +34,7 @@ const (
 	WaitMedium              = 3 * time.Minute
 	WaitLong                = 15 * time.Minute
 	RetryMedium             = 5 * time.Second
+	RetryShort              = 1 * time.Second
```

Comment:
Small one, I would reorder these for consistency with the above ones for waiting
```suggestion
	RetryShort              = 1 * time.Second
	RetryMedium             = 5 * time.Second
```

Reply:
Fixed